### PR TITLE
fix(builder-image-generator): Fix up checks from previous manual merge

### PR DESCRIPTION
**Pull request by jimmidyson**
These failed automatic checks in #688 but that was ignored and manually merged, breaking all future PRs...

### CODE REVIEW

File: syndesis-builder-image-generator/src/main/java/io/syndesis/image/Application.java [hunk 1]
```diff
@@ -60,18 +61,19 @@ public static void main(String[] args) {
...
-            System.out.println("To: "+to);
+            System.out.println("To: "+to); //NOPMD
             generateIntegrationProject(new File(to));
-            System.exit(0);
         } catch (IOException e) {
-            System.exit(1);
+            e.printStackTrace(); //NOPMD
+            System.exit(1); //NOPMD
         }
     }
```
Comment (zregvart) [inline syndesis-builder-image-generator/src/main/java/io/syndesis/image/Application.java]: Should we reimplement this as [CommandLineRunner](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-spring-application.html#boot-features-command-line-runner), happy to have it in another PR.

Comment (jimmidyson) [reply syndesis-builder-image-generator/src/main/java/io/syndesis/image/Application.java]: Good idea. I will look at that soon, after finishing off the keycloak removal stuff now.

**Comment by jimmidyson** [comment]
Would be good to get review approval @zregvart. Thanks!

**Comment by zregvart** [comment]
Also see https://github.com/syndesisio/syndesis-rest/issues/693

File: syndesis-builder-image-generator/src/main/java/io/syndesis/image/Application.java [hunk 2]
```diff
@@ -140,16 +143,17 @@ static private void generate(GenerateProjectRequest request, File targetDir) thr
...
+            }
             os.flush();
             return os.toByteArray();
         }
     }
 
 
     // Helper method to help construct maps with concise syntax
-    static private Map<String, String> map(Object... values) {
+    private static Map<String, String> map(Object... values) {
```
Comment (rhuss) [inline syndesis-builder-image-generator/src/main/java/io/syndesis/image/Application.java]: ha !

**Pull request merged**